client/src/newStore.js

```diff
@@ -0,0 +1,25 @@
+import { applyMiddleware, compose as reduxCompose, createStore } from 'redux';
```

Comment:
@martinjoha Btw I don't see this file being used anywhere, can I just remove it?

Reply:
its fine to delete it